src/webui/.eslintrc

```diff
@@ -5,6 +5,9 @@
     "jest": true,
     "es6": true
   },
+  "globals": {
```

Comment:
What does it do? 🤔

Reply:
I meant, the `"__DEBUG__": true`

Reply:
Only set `__webpack_public_path__` in production mode, it will `react-hot-loader`